extensions/http-curl/tests/integration/C2NullConfiguration.cpp

```diff
@@ -29,26 +29,27 @@
 #include <vector>
 #include <iostream>
 #include <sstream>
-#include "HTTPClient.h"
-#include "InvokeHTTP.h"
-#include "../TestBase.h"
+#include "client/HTTPClient.h"
+#include "processors/InvokeHTTP.h"
 #include "utils/StringUtils.h"
 #include "core/Core.h"
-#include "../include/core/logging/Logger.h"
```

Comment:
oh man. everytime I see these I get upset. Eclipse does this to me. I use eclipse cdt and intellij for java....but never can understand why eclipse does this on a fresh workspace.